homeassistant/components/google_sheets/strings.json

```
@@ -45,6 +45,10 @@
     "append_sheet": {
       "description": "Appends data to a worksheet in Google Sheets.",
       "fields": {
+        "add_created_column": {
+          "description": "Include \"created\" column containing current date-time to the data being appended.",
```

```suggestion
          "description": "Add a \"created\" column with the current date-time to the appended data.",
```

To make it a bit more natural.